test/NameRegistry.t.sol

```diff
@@ -178,6 +179,7 @@ contract NameRegistryTest is Test {
 
     function testMakeCommitAfterReplayDelay(address alice, bytes32 secret, address recovery, uint256 delay) public {
         _disableTrusted();
+        delay = delay % REASONABLE_SLA_PERIOD; // test delays up to REASONABLE_SLA_PERIOD && protect against overflow
```

Comment:
comment can be removed, we don't add it anywhere else and i think the behavior is somewhat understandable to the average solidity dev